x-pack/plugins/observability_solution/apm/server/routes/correlations/queries/fetch_duration_histogram_range_steps.ts

```diff
@@ -100,8 +104,9 @@ export const fetchDurationHistogramRangeSteps = async ({
     return { rangeSteps: [] };
   }
 
-  const durationMin = resp.aggregations.duration_min.value;
-  const durationMax = resp.aggregations.duration_max.value * 2;
+  // these values should never be 0, so if they are we set them to 1
+  const durationMin = Math.min(1, resp.aggregations.duration_min.value);
```

Comment:
I think here you need `Math.max` . `Math.min(1, 0)` returns 0 right?

Reply:
100%

Reply:
totally right, thanks for the review @kpatticha